### ISSUE TITLE
Have a global option to change the temporary directory

### ISSUE THREAD

**Issue by privefl** [HenrikBengtsson/Wishlist-for-R#132]
```r
options(tmpdir = ".")
tmpfile()
# ./fileXXXXXXXX
```

**Comment by nfultz** [comment]
There's an environment variable for this, I've had to use it to install packages on red hat when the default temp location did not have execute privileges. I think it has to be an (exported) environment variable so that subprocesses can also see it.

**Comment by privefl** [comment]
Yes, I know, but this means you have to start new R subprocesses, which is less straightforward than just setting a global option.
